presto-main/src/test/java/io/prestosql/sql/planner/TestPredicatePushdown.java

```diff
@@ -520,9 +521,55 @@ public void testTablePredicateIsExtracted()
                                                 "orders",
                                                 ImmutableMap.of("ORDERSTATUS", "orderstatus"))),
                                 anyTree(
-                                        filter("CAST(NAME AS varchar(1)) IN ('F', 'O')",
+                                        filter("CAST(NAME AS varchar(1)) IN ('F', 'O') AND (NOT (CAST(NAME AS VARCHAR(1))) IS NULL)",
```

Comment:
`(NOT (CAST(NAME AS VARCHAR(1))) IS NULL`

Reply:
I'll be handling them in a follow-up PR